updater/spec/fixtures/bundler_grouped_by_types/updated_development_deps/Gemfile.lock

```diff
@@ -32,7 +32,7 @@ GEM
     ruby-progressbar (1.13.0)
     toml-rb (2.2.0)
       citrus (~> 3.0, > 3.0)
-    unicode-display_width (2.4.2)
+    unicode-display_width (2.5.0)
```

Comment:
❓ Is there any context around why this was bumped?

Reply:
Did you see the commit message that bumped it? A new version was released, this is unrelated to the PR but I didn't want to go through the trouble of a separate PR to address it to geen CI green here

Reply:
Ah! I had only looked at the diff 😅 Makes sense!